docker/webui/run_openqa.sh

```diff
@@ -7,9 +7,36 @@ else
   /usr/share/openqa/script/upgradedb --user geekotest --upgrade_database
```

Comment:
Have you tested whether you can get rid of these "prepare environment" steps? The check for the SQLite database is definitely outdated. Since you're extending this outdated file you should make sure that all of its code makes actually still sense. I suppose the first daemon which make use of the database will perform the necessary steps to initialize/update the database so you can likely simply remove this "prepare environment" code completely.

Reply:
You are right, this have to be removed